_episodes/dataflow.md

```diff
@@ -17,3 +17,192 @@ learners should be able to
 __explain how a workflow document describes the input and output of a workflow and the flow of data between tools__
 
 {% include links.md %}
+
+
+
+CWL Workflows are about chaining different steps.
+
+A workflow is the orchestration of the individual steps.
```

Comment:
Are we talking about workflows in general, or CWL workflows.  If we are talking about workflows in general,  I would put this statement first.  If we are talking about CWL workflows, I think this is a good opportunity to explain what I CWL step is and what they can contain - and talk about connecting tools together.    This definitely might be too formal "A workflow is a process characterized by multiple subprocess steps, where step outputs are connected to the inputs of downstream steps to form a directed acylic graph, and independent steps may run concurrently."   But something that mentions steps are subprocesses and the inputs are connected to downstream steps.

Reply:
If we are talking about workflows in general --- I think the description here is quite good and maybe we could model our description after it: https://direct.mit.edu/dint/article/2/1-2/108/10003/FAIR-Computational-Workflows

"Generally speaking, a workflow is a precise description of a procedure – a multi-step process to coordinate multiple tasks and their data dependencies. In computational workflows each task represents the execution of a computational process, such as: running a code, the invocation of a service, the calling of a command line tool, access to a database, submission of a job to a compute cloud, or the execution of data processing script or workflow."